modules/packages/zypper.in

```diff
@@ -362,14 +371,14 @@ def repo_install():
 
             cmd_line += block
 
-            subprocess_call(cmd_line, stdout=NULLFILE)
+            subprocess_call(cmd_line, stdout=NULLFILE, stderr=subprocess.PIPE)
 
             # No final check. CFEngine will figure out that it's missing
             # if it failed.
 
-    # ret == 0 doesn't mean we succeeded with everything, but it's expensive to
+    # return 0 doesn't mean we succeeded with everything, but it's expensive to
     # check, so let CFEngine do that.
-    return ret
+    return 0
```

Comment:
Why did you change these to always return 0 ? If that is really necessary, you have to explain why in the commit message.